### PR TITLE
Bug 1458312 - [e2e-tests] Un-pin dependencies in Pipfile; use Pipfile.lock w/pyup, instead

### PR DESCRIPTION
**Pull request by stephendonner**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1458312 for more details, and some questions we should answer!

Passing adhoc build here: https://gist.github.com/stephendonner/c418f60eee4556141d69557331bdfe99

### CODE REVIEW

**Comment by stephendonner** [comment]
@davehunt thanks; I (hopefully obviously) wasn't sure which direction to head, so I'll re-route to do that.

**Comment by stephendonner** [comment]
OK, the latest effort:
* unpins dependencies in Pipfile
* updates Pipfile.lock's hash (auto-generated)
* only has @pyup-bot update Pipfile.lock

Passing ad-hoc here: https://gist.github.com/stephendonner/a5548994d9a5f1ac1b43913682390665

File: e2e-tests/Pipfile [hunk 1]
```diff
@@ -11,15 +11,15 @@ name = "pypi"
...
+pytest = "*"
+pytest-base-url = "*"
+pytest-html = "*"
+pytest-metadata = "*"
+pytest-selenium = "*"
+pytest-variables = "*"
+pytest-xdist = "*"
+selenium = "*"
+requests = "*"
+pipenv = "*"
```
[the remark this answers is not in the document]
Comment (stephendonner) [reply e2e-tests/Pipfile]: No; this was auto-generated, and I was careful/hesitant/lazy, so didn't remove it, before.

I'll do so now.

**Comment by stephendonner** [comment]
@Osmose @willkg this is ready for another, hopefully-final r? pass + merge - thanks!

**Comment by Osmose** [comment]
If @davehunt is happy I'm happy

**Comment by stephendonner** [comment]
This...broke our e2e-tests build, due to sha-hash mismatch in the Pipfile.lock, which I tested and submitted a fix for over in https://github.com/mozilla-services/socorro/pull/4430

**Pull request merged**